classes/MyTableLister.php

```diff
@@ -628,6 +638,7 @@ public function pagination($rowsPerPage, $totalRows, $offset = null, $options =
                 $output .= $this->addPage($currentPage - 1, $currentPage, $rowsPerPage, $this->translate('Previous'), $title);
             }
             for ($page = 1; $page <= $pages; $page++) {
+                // TODO ask CRS2 Method GodsDev\MyCMS\MyTableLister::addPage() invoked with 6 parameters, 3-5 required.
                 $output .= $this->addPage($page, $currentPage, $rowsPerPage, null, $this->translate('Go to page'), $title);
```

Comment:
@crs2 Ten 6.parametr $title je zřejmě zbytečně použit, neboť addPage akceptuje 3-5 parametrů?

Reply:
Tady prosím vyhodit ", $this->translate('Go to page')" a nechat jako poslední dva parametry: null, $title